portality/templates/application_form/editor_application.html

```diff
@@ -7,32 +7,13 @@
 {% set js_validation = True %}
 {% set auto_save = 0 %}
 {% set notabs = true %}
+{% set diff_table = false %}
```

Comment:
This file only only differs from the associate form by the form_id - can we further deduplicate these files by handling these variables in the view by passing in a config object?)